backend/spec/spec_helper.rb

```diff
@@ -107,12 +108,12 @@
 
   config.include FactoryBot::Syntax::Methods
   config.include ActiveSupport::Testing::Assertions
```

Comment:
I think that without `ActiveJob::TestHelper` we can get rid of `ActiveSupport::Testing::Assertions` too